packages/gestalt/src/InternalLink.js

```diff
@@ -127,6 +130,7 @@ const InternalLinkWithForwardRef: AbstractComponent<Props, HTMLAnchorElement> =
       : {},
     isTapArea
       ? {
+          [layoutStyles.block]: true,
```

Comment:
TapArea has no inline. Moving here from line 107
![image](https://user-images.githubusercontent.com/10593890/116452313-98781a00-a82b-11eb-8b50-5cf54c1c620b.png)

Reply:
Totally fine since we're matching current behavior. Should we include an inline prop in the future? Is it needed?

Reply:
I don't know, we should evaluate that.